src/shared/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+class Utils {
+
+    static formatDate(dateStr: string): string {
+        const date = new Date(dateStr);
+
+        return moment(date).format("YYYY MM DD");
+    }
+
+    static formatGender(gender: string): string {
+        return gender.includes('F') ? 'Female' : 'Male';
+    }
+
+    static calculatePatientAge(patientDOB: string): number {
+        const today = new Date();
+        const dob = new Date(patientDOB);
+        return today.getFullYear() - dob.getFullYear();
```

Comment:
This logic might result in incorrect age since it does not take into account the month and date when the person was born.

As an example, if one was born on 31 December 2020, meaning one is ~3 months old, the above function would return 1 year old.

```suggestion
        return moment().diff(patientDOB, 'year');
```